stream-chat-android-client/src/main/java/io/getstream/chat/android/client/ChatClient.kt

```diff
@@ -720,6 +721,10 @@ public class ChatClient internal constructor(
      * @param channelType the channel type. ie messaging
      * @param channelId the channel id. ie 123
      */
+    @Deprecated(
+        message = "Use getChannelClient() instead",
+        replaceWith = ReplaceWith("this.getChannelClient")
+    )
```

Comment:
I was thinking about it but it will be a breaking change for our customers.
Providing a new method will give them more time to migrate from `ChannelController` to `ChannelClient`

Reply:
It's not breaking if `ChannelClient` is a `ChannelController`, is it?

Reply:
ah I can see now. It makes sense when `ChannelClient` implements`ChannelController`. I'll do that